templates/OperationResponseNoBodySnippet.stencil

```diff
@@ -1,8 +1,20 @@
 switch result {
-case .success:
-    DispatchQueue.main.async {
-        completionHandler(.success(()), httpResponse)
+case let .success(data):
+    {% if op.exceptions.count > 0 %}
+    {% include "OperationResponseData.stencil" %}
+    {% endif %}
+    let statusCode = httpResponse?.statusCode ?? -1
+    if [
+    {% for statusCode in op.returnType.statusCodes %}
+    {{ statusCode }},
+    {% endfor  %}
+    ].contains(statusCode)
```

Comment:
These five lines of code seem to be reused a lot. Would it makes sense to put these in a snippet or do you think it makes it harder to reason about the logic?

Reply:
Refactor into OperationResponseStatusCodeSnippet.stencil